engine/gamesys/src/gamesys/components/comp_sprite.cpp

```diff
@@ -830,6 +829,10 @@ namespace dmGameSystem
                 Matrix4 world = dmGameObject::GetWorldMatrix(c->m_Instance);
                 Vector3 size( c->m_Size.getX() * c->m_Scale.getX(), c->m_Size.getY() * c->m_Scale.getY(), 1);
                 c->m_World = appendScale(world * local, size);
+                // we need to consider the full scale here
+                // I.e. we want the length of the diagonal C, where C = X + Y
+                float radius_sq = Vectormath::Aos::lengthSqr((c->m_World.getCol(0).getXYZ() + c->m_World.getCol(1).getXYZ()) * 0.5f);
+                sprite_world->m_BoundingVolumes[i] = radius_sq;
```

Comment:
One benefit of doing the calculation here is that it's only done once per frame, and not once per draw call.

Also, removed some extra calculations by taking the length from the diagonal, as opposed to doing it individually per axis.

Reply:
👍